### PR TITLE
fix the backend validation requirement for middle name to not be empty

### PR DESCRIPTION
**Pull request by heaversm**
plus

fix a layout issue on the modal trigger for birth year
move the "cancel service" button to the form confirm screen and out of the form itself.

### CODE REVIEW

File: views/partials/dashboards/remove-form.hbs
Comment (pdehaan) [inline views/partials/dashboards/remove-form.hbs:115]: Should the `alt` be localized?

**Review by toufali — APPROVED** [review]
My local environment is not set up to test this in-browser, so I'll wait until after merge.

File: views/partials/dashboards/remove-form.hbs
Comment (heaversm) [reply views/partials/dashboards/remove-form.hbs:115]: I'll put it in strings to be consistent - good catch! (none of the `getRemoveString` function calls actually get localized, but it keeps that possibility open).

**Pull request merged**
